src/Welcome.js

```diff
@@ -43,8 +44,9 @@ const Welcome = () => {
   };
 
   return (
-    <div>
-      <p>Welcome to your smart shopping list!</p>
+    <div className="welcome">
+      <h1>Orange You Glad</h1>
+      <h2>you made a list</h2>
```

Comment:
Semantically, I'm not totally sure it makes sense to have "you made a list" as part of an `h2` rather than still being part of the `h1` but stylistically set below it. I think it might be best to have "Orange you glad you made a list" all part of the `h1`, but wrap "you made a list" in some kind of block styling to set it below the main title. Think: when a screen reader reads this, will it make sense it it is broken up?

Reply:
I agree with you. Would we put "you made a list" inside a span, so it looks like 
`<h1>Orange you glad <span>you made a list</span></h1>` and then target that span element to have the font size we desire in css?